experiments/generic/cpu_hog/cpu_hog_ansible_logic.yml

```diff
@@ -0,0 +1,93 @@
+---
+- hosts: localhost
+  connection: local  
+
+  vars:
+    c_experiment: "cpu-hog"
+    c_duration: "{{ lookup('env','TOTAL_CHAOS_DURATION') }}"
+    a_ns: "{{ lookup('env','APP_NAMESPACE') }}"
+    a_label: "{{ lookup('env','APP_LABEL') }}"
+    a_kind: "{{ lookup('env','APP_KIND') }}"
+    a_platform: "{{ lookup('env', 'PLATFORM') }}"
+
+  tasks:
+
+    - block:
+      
+        - include: cpu_hog_ansible_prerequisites.yml
+
+        - include_vars:
+            file: chaosutil.yml
+            
+        ## GENERATE EXP RESULT NAME
+        - block:
+            - name: Construct chaos result name (experiment_name)
+              set_fact:
+                c_experiment: "{{ lookup('env','CHAOSENGINE') }}-{{ c_experiment }}"
+          when: lookup('env','CHAOSENGINE')
+
+        ## RECORD START-OF-EXPERIMENT IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'SOT'
+            namespace: "{{ a_ns }}"
+
+        ## PRE-CHAOS APPLICATION LIVENESS CHECK
+        - name: Verify that the AUT (Application Under Test) is running
+          include_tasks: "/utils/common/status_app_pod.yml"
+          vars:
+            app_ns: "{{ a_ns }}"
+            app_lkey: "{{ a_label.split('=')[0] }}"
+            app_lvalue: "{{ a_label.split('=')[1] }}"
+            delay: 5
+            retries: 60 
+
+         ## READY TO GIVE CPU HOG 
+        - name: Ready to give cpu-hog
+          include_tasks: "{{ c_util }}"
+
+        ## POST-CHAOS APPLICATION LIVENESS CHECK
+        - name: Verify AUT liveness post fault-injection
+          include_tasks: "/utils/common/status_app_pod.yml"
+          vars:
+            app_ns: "{{ a_ns }}"
+            app_lkey: "{{ a_label.split('=')[0] }}"
+            app_lvalue: "{{ a_label.split('=')[1] }}"
+            delay: 2
+            retries: 150
+          when: node_status.stdout == 'Ready'
+
+        ## POST-CHAOS APPLICATION LIVENESS CHECK
+        - name: Verify AUT liveness post fault-injection
+          include_tasks: "/utils/common/status_app_pod.yml"
+          vars:
+            app_ns: "{{ a_ns }}"
+            app_lkey: "{{ a_label.split('=')[0] }}"
+            app_lvalue: "{{ a_label.split('=')[1] }}"
+            delay: 2
+            retries: 150
+          ignore_errors: yes
+          when: node_status.stdout != 'Ready'
+
+        - set_fact:
+            flag: "Pass"
+
+      rescue:
+         
+        - set_fact:
+            flag: "Fail"
+    
+      always:
+
+        ## RECORD END-OF-TEST IN LITMUSCHAOS RESULT CR
+        - include_tasks: /utils/runtime/update_chaos_result_resource.yml
+          vars:
+            status: 'EOT'
+            namespace: "{{ a_ns }}"
+
+
+
+
+
+
+
```

Comment:
Can you remove this many empty lines from the last of file just one line is enough, You can do this in next PRs